docs/references/remix/spa-mode.mdx

```diff
@@ -43,6 +66,52 @@ function App() {
   );
 }
 
+export default ClerkApp(App, {
+  publishableKey: 'pk_test_XYZ',
```

Comment:
Does this not read the `CLERK_PUBLISHABLE_KEY` environment variable the way the SSR version does?

Reply:
@royanger Currently, no. I will research in a later release, if we can leverage Vite's `import.meta.env`.

Reply:
Also, this would require users to use the `VITE_` prefixed variables. 

Do you think it would be confusing?

Reply:
@royanger Unfortunately, after investigating a bit, the `import.meta` is not available in CommonJS modules, which we still support for Remix, so it's not an easy win to support this. We will revisit this in the future, maybe when we drop commonjs support in a future major version.

Reply:
I decided to change it and provide a similar code snippet as we do in our React quickstart, where we set the `.env` file and then we suggest the users uses the `import.meta` themselves in their `app/root.tsx` file. I think this is a good middleground for now, and we will revisit in the future.